include/CommandHandler/src/CommandHandler.cpp

```diff
@@ -250,26 +256,26 @@ void CommandHandler::HandleHelp(SocketWrapper& socket_wrapper)
         "HELO, MAIL FROM, RCPT TO, DATA, "
         "QUIT, NOOP, RSET, VRFY, HELP, "
         "STARTTLS, AUTH PLAIN, REGISTER\r\n";
-    try {
-        socket_wrapper.SendResponseAsync(
-            "214 The following commands are recognized: " + supported_commands
-        ).get();
+    try
+    {
+        socket_wrapper.SendResponseAsync("214 The following commands are recognized: " + supported_commands).get();
         Logger::LogProd("CommandHandler::HandleHelp: Successfully sent HELP response to client.");
-    } catch (const std::exception& e) {
+    }
+    catch (const std::exception& e)
+    {
         Logger::LogError("CommandHandler::HandleHelp: Exception caught while sending HELP response: " +
-            std::string(e.what()));
+                         std::string(e.what()));
         ErrorHandler::HandleError("Handle HELP", e, socket_wrapper, "550 Internal Server Error\r\n");
     }
     Logger::LogDebug("Exiting CommandHandler::HandleHelp");
 }
 
-
 void CommandHandler::HandleVrfy(SocketWrapper& socket_wrapper, const std::string& line)
 {
     Logger::LogDebug("Entering CommandHandler::HandleVrfy");
     Logger::LogTrace("CommandHandler::HandleVrfy parameter: SocketWrapper reference");
 
-    const std::string user_name = line.substr(5);
+    const std::string user_name = line.substr(USERNAME_START_INDEX);
     Logger::LogTrace("VRFY command received for user: " + user_name);
 
```

Comment:
Should add checking if user_name is empty, maybe, you need to check if user_name is syntactically valid